jib-core/src/integration-test/java/com/google/cloud/tools/jib/registry/LocalRegistry.java

```diff
@@ -91,7 +93,7 @@ public void start() throws IOException, InterruptedException {
               "-e",
               "REGISTRY_AUTH_HTPASSWD_PATH=/auth/htpasswd"));
     }
-    dockerTokens.add("registry:2");
+    dockerTokens.add("registry.ipv6.docker.com/library/registry:2");
```

Comment:
I guess this change shouldn't be necessary?

Reply:
hmm you might be right! Let me try removing this and see what happens.